### DISCUSSION TITLE
Unable to update my mod.

### DISCUSSION THREAD

**Discussion by 12122234** [FabricMC/fabric-api#2959] [Mod Dev Support]
As the title says I have tried to update my mod but failed, I have tried every way and every build.
I updated the gradle.properties file but it made no effect.

**Comment by Daomephsta** [comment]
It's not possible to help with so little information. 
Attempt to update it again, record what you do in detail, and what the issues are.
Once we know what you did and what issues you encountered, then we can help.

**Reply by 12122234** [reply]
The gradle.properties file has the values I need. I use VSCode, and the tutorials I watch that use IntelliJ Idea do some builds after updating the gradle.properties file, and because I just can't find those builds, I just ignore that part, and after updating the mod's files I find out that the code is still recognizing that it's the old version, and opening minecraft makes it open as the old one. And that's how I made this post.

**Comment by alazeprt** [comment]
Have you modified the dependency values in fabric.mod.json?
